next.config.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
+    images: {
+        loader: 'imgix',
+        path: 'https://noop/'
+    },
```

Comment:
Does this affect images in general?

Reply:
I've been reading and so far, it doesn't seem like it's going to affect the images. I took this solution from a problem they had [here](https://github.com/vercel/next.js/issues/21079)